tools/crm_mon.c

```diff
@@ -138,6 +138,7 @@ crm_trigger_t *refresh_trigger = NULL;
 
 /* Define exit codes for monitoring-compatible output */
 #define MON_STATUS_WARN CRM_EX_ERROR
+#define MON_STATUS_ERR  CRM_EX_ERROR
```

Comment:
For nagios plugins, the possibilities are OK=0, WARN=1, CRIT=2, and UNKNOWN=3. Defining MON_STATUS_WARN to CRM_EX_ERROR was a questionable choice, but I chose it so it would map to one of the new enum values for use with functions that took the enum as an argument. We can define MON_STATUS_CRIT to either 2 or CRM_EX_INVALID_PARAM.

Reply:
Guess using CRM_EX_INVALID_PARAM plus a little bit comment mentioning the mapping to nagios plugin exits would be the right choice then.